doc/tutorials/enable_ivshmem.rst

```diff
@@ -0,0 +1,166 @@
+.. _enable_ivshmem:
+
+Enable Inter-VM Communication Based on Ivshmem
+##############################################
+
+You can use inter-vm communication based on the ``ivshmem`` dm-land
+solution or hv-land solution, according to the usage scenario needs.
+(See :ref:`ivshmem-hld` for a high-level description of these solutions.)
+While both solutions can be used at the same time, VMs using different
+solutions can not communicate with each other.
+
+ivshmem dm-land usage
+*********************
+
+Add this line as an ``acrn-dm`` boot parameter::
+
+     -s slot,ivshmem,shm_name,shm_size
+
+where
+
+-  ``-s slot``  - Specify the virtual PCI slot number
+
+-  ``ivshmem``  - Virtual PCI device name
+
+-  ``shm_name`` - Specify a shared memory name. Post-launched VMs with the same
+   ``shm_name`` share a shared memory region. The ``shm_name`` needs to start
+   with ``dm:/`` prefix. For example, ``dm:/test``
+
+-  ``shm_size`` - Specify a shared memory size. The unit is byte. The size
+   ranges from 2 megabytes to 512 megabytes and must be a power of 2.
+   For example, Set up a shared memory of 2 megabytes, use ``2097152``
+   instead of ``shm_size``. The two communicating VMs must define the same size.
+
+.. note:: This device can be used with Real-Time VM (RTVM) as well.
+
+ivshmem hv-land usage
+*********************
+
+The ``ivshmem`` hv-land solution is disabled by default in ACRN. You
+enable it using the  :ref:`acrn_configuration_tool` with these steps:
+
+- Enable ``ivshmem`` hv-land in ACRN XML configuration file. For example, the
+  XML configuration file for the hybrid_rt scenario on a whl-ipc-i5 board is found in
+  ``acrn-hypervisor/misc/vm_configs/xmls/config-xmls/whl-ipc-i5/hybrid_rt.xml``
+
+   - Configure ``IVSHMEM_ENABLED`` to ``y`` in ACRN scenario XML configuration
```

Comment:
Let's clarify this by writing it the same as you did for the last example:
```
   - Edit XML configuration file for hybrid_rt scenario on whl-ipc-i5 board
     ``acrn-hypervisor/misc/vm_configs/xmls/config-xmls/whl-ipc-i5/hybrid_rt.xml``
     to enable ``ivshmem`` and configure the shared memory region using the format
     ``shm_name, shm_size, VM IDs`` (as described above in the ACRN dm boot parameters).
     The region name must start with ``hv:/`` for an hv-land shared region, and we'll allocate 2MB
     shared between VMs 0 and 2:

     .. code-block:: none
        :emphasize-lines: 2,3

        <IVSHMEM desc="IVSHMEM configuration">
		<IVSHMEM_ENABLED>y</IVSHMEM_ENABLED>
		<IVSHMEM_REGION>hv:/shm_region_0, 0x200000, 0:2</IVSHMEM_REGION>
	</IVSHMEM>
```

Reply:
Got it ,thanks